src/sap/fhir/model/r4/FHIRFilterOperator.js

```diff
@@ -41,7 +41,49 @@ sap.ui.define(["sap/ui/model/FilterOperator"], function(FilterOperator) {
 		 *
 		 * @public
 		 */
-		AP: "ap"
+		AP: "ap",
+		/**
+		 * The set is empty or not (value is false or true)
```

Comment:
are the new introduced operators also available for simple filtering?

Reply:
if not, i would recommend to create a new `FHIRFilterComplexOperator.js` which extends the `FHIRFilterOperator`

Reply:
done